src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -7,8 +7,11 @@
 var $ = require( 'jquery' );
 var handlebars = require( 'handlebars' );
 
-var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" style="display: none;">' +
-  '<span class="cf-notification_icon cf-notification_icon__{{ state }} cf-icon cf-icon-{{ icon }}-round"></span>' +
+var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" ' +
+                             'style="display: none;">' +
+  '<span class="cf-notification_icon ' +
```

Comment:
Hm, okay, it's fine for now I think, but we may want to find ways to have the templates in separate .tmpl files in the future.